test/spec/unit/pbjs_api_spec.js

```diff
@@ -812,7 +812,114 @@ describe('Unit: Prebid Module', function () {
     });
   });
 
+  describe('temp', () => {
```

Comment:
is this meant to still be here?  If so, it probably deserves a better name than 'temp'

Reply:
Yes it was meant to be there. Updated the describe.